src/hardware/mame/saa1099.cpp

```diff
@@ -159,7 +159,7 @@ saa1099_device::saa1099_device(const machine_config &mconfig,
           m_sync_state(0),
           m_selected_reg(0),
           m_sample_rate(0.0),
-          m_master_clock(0)
+          m_baseline_clock(0)
```

Comment:
Not sure if "baseline" is good name in here… It'll have the same value as the one used in commit before (double check it, please).

Reply:
I agree.

Given this is the chip's clock speed, I think calling it exactly like that is even clearer: `m_chip_clock` .

Reply:
Now that I've made this change, I see it falls inline with the approach I used for the Innovation SSI-2001:

![2021-07-08_13-00](https://user-images.githubusercontent.com/1557255/124983122-71d51d00-dfec-11eb-882e-b83a65fadb3e.png)